bin/src/command/orders.rs

```diff
@@ -529,7 +529,10 @@ impl CommandServer {
             fork_main_into_new_main(self.executable_path.clone(), upgrade_data)
                 .with_context(|| "Could not start a new main process")?;
 
-        fork_confirmation_channel.blocking();
+        // DISCUSS: should we propagate the error instead of printing it?
+        if let Err(e) = fork_confirmation_channel.blocking() {
```

Comment:
To me, it is not necessary, but the error message should prevent the user that something has failed and it is bad, it may need to restart sozu

Reply:
I need to dig in, but we should know, if something is going wrong on the worker side or not. I will be back :)